SPECS-EXTENDED/rust-cbindgen/rust-cbindgen.spec

```
@@ -0,0 +1,243 @@
+# Generated by rust2rpm 24
+# * tests are completely broken with Cython 3.0.0 on Fedora 39+
+Summary:        Tool for generating C bindings to Rust code
+Name:           rust-cbindgen
+Version:        0.24.3
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/mozilla/cbindgen
+Source:         %{url}/archive/refs/tags/v0.24.3.tar.gz
```

The format for source0 should be:
`%{name}-%{version}.ext`
In this case it should look like:
```suggestion
Source:         %{url}/archive/refs/tags/v0.24.3.tar.gz#/%{name}-%{version}.tar.gz
```
This would also mean a corresponding update of `signatures.json`

---

We might have missed to update the signature file